src/NosCore.Data/Resource/LocalizedResources.ru.resx

```diff
@@ -117,4 +117,636 @@
+  </data>
+  <data name="MAPNPCS_LOADED" xml:space="preserve">
+    <value>Загружено {0} MapNpcs!</value>
+  </data>
+  <data name="MAPS_LOADED" xml:space="preserve">
+    <value>Загружено {0} Maps!</value>
+  </data>
+  <data name="MAPS_PARSED" xml:space="preserve">
+    <value>Загружено {0} Map!</value>
+  </data>
+  <data name="MAPTYPES_PARSED" xml:space="preserve">
+    <value>Загружено {0} MapTypes!</value>
+  </data>
+  <data name="MAP_DONT_EXIST" xml:space="preserve">
+    <value>Эта локация не существует!</value>
+  </data>
+  <data name="MARTIALARTIST" xml:space="preserve">
+    <value>Мастер боевых искусств</value>
+  </data>
+  <data name="MASTER_SERVER_PING" xml:space="preserve">
+    <value>Пингуем Master сервер</value>
+    <comment>нужно думать, как будет грамотнее, использовать Master или локализовать как Главный сервер?</comment>
+  </data>
+  <data name="MASTER_SERVER_PING_FAILED" xml:space="preserve">
+    <value>Не удалось пропинговать Master сервер</value>
+    <comment>нужно думать, как будет грамотнее, использовать Master или локализовать как Главный сервер?</comment>
+  </data>
+  <data name="MASTER_SERVER_RETRY" xml:space="preserve">
+    <value>Не удалось подключиться к Master серверу! Повторная попытка через {0} секунд...</value>
+    <comment>нужно думать, как будет грамотнее, использовать Master или локализовать как Главный сервер?</comment>
+  </data>
+  <data name="MAX_GOLD" xml:space="preserve">
+    <value>Достигнуто максимальное количество золота!</value>
+  </data>
+  <data name="MONSTERS_PARSED" xml:space="preserve">
+    <value>Импортировано {0} MapMonsters!</value>
+  </data>
+  <data name="NEW_LEADER" xml:space="preserve">
+    <value>Теперь вы новый глава группы!</value>
+  </data>
+  <data name="NOT_ADVENTURER" xml:space="preserve">
+    <value>Вы не Искатель приключений.</value>
+  </data>
+  <data name="NOT_ENOUGH_MONEY" xml:space="preserve">
+    <value>Недостаточно денег для покупки предмета.</value>
+  </data>
+  <data name="NOT_ENOUGH_PLACE" xml:space="preserve">
+    <value>Места больше нет.</value>
+  </data>
+  <data name="NOT_ENOUGH_REPUT" xml:space="preserve">
+    <value>Недостаточно репутации для покупки предмета.</value>
+  </data>
+  <data name="NOT_IN_BLACKLIST" xml:space="preserve">
+    <value>Этот игрок не находится в Вашем черном списке!</value>
+  </data>
+  <data name="NOT_YOUR_ITEM" xml:space="preserve">
+    <value>Этот предмет не принадлежит Вам!</value>
+  </data>
+  <data name="NO_ITEM" xml:space="preserve">
+    <value>Предмет не существует!</value>
+    <comment>или отсутствует?</comment>
+  </data>
+  <data name="NO_MAP" xml:space="preserve">
+    <value>Для работы сервера необходимо установить хотя бы карту НосВилля.</value>
+  </data>
+  <data name="NO_SP" xml:space="preserve">
+    <value>Не экипирована Карта специалиста.</value>
+  </data>
+  <data name="NO_SPECIAL_PROPERTIES_WEARABLE" xml:space="preserve">
+    <value>No special properties wearable</value>
```

Comment:
if it's not translated don't add it

Reply:
Some entries not translated. Need to delete it?

Reply:
yes